plugin-statusnotifier/statusnotifierbutton.h

```diff
@@ -79,6 +92,10 @@ public slots:
 
     ILXQtPanelPlugin* mPlugin;
 
+    QString mTitle;
+    bool mAutoHide;
+    QTimer mHideTimer;
```

Comment:
Better to use a different name than mHideTimer in StatusNotifierWidget.

Reply:
Different tastes :)